code/html/custom.js

```diff
@@ -459,6 +459,30 @@ function doRestore() {
     return false;
 }
 
+function doFactoryReset() {
+    var response;
+
+    ask = (typeof ask == "undefined") ? true : ask;
+
+    if (numChanged > 0) {
```

Comment:
Unnecessary, you are about to delete all settings with the factory reset, there is no need to save changes before.